### DISCUSSION TITLE
Increase Chart Height to Show all Labels

### DISCUSSION THREAD

**Discussion by bennettblack** [filamentphp/filament#6002] [Help]
With the following bar-chart as an example, is it possible to increase the height of the chart to allow for more labels to be seen? At present, all bars are shown, but not all labels. 

I've tried overriding the `maxHeight` property, but I don't think that's the solution to this issue. 

![image](https://user-images.githubusercontent.com/51721783/227042462-8888c168-87c2-475f-a943-fda19a23abde.png)

**Comment by danharrin** [comment]
Have you found a way to do this with Chart.js itself?

**Reply by bennettblack** [reply]
Hey Dan,

I found that if you add `'maintainAspectRatio' => false` to the chart options, you can set the Canvas height as desired (in Browser DevTools). I'm struggling with how best to do that - since I have multiple charts on the same dashboard, and they don't have ID attributes for easy selection. 

The only thing I can think of is overriding the view Filement uses (`chart-widget.blade`) and adding a method to set height in the style tag of the Canvas element.

**Reply by danharrin** [reply]
You could submit a PR to add something similar to `getMaxHeight()`? Overriding views is not ideal

**Reply by bennettblack** [reply]
Absolutely - will do!
